fe/fe-core/src/main/java/org/apache/doris/datasource/property/constants/S3Properties.java

```diff
@@ -334,6 +335,18 @@ public static Cloud.ObjectStoreInfoPB.Builder getObjStoreInfoPB(Map<String, Stri
             // S3 Provider properties should be case insensitive.
             builder.setProvider(Provider.valueOf(properties.get(S3Properties.PROVIDER).toUpperCase()));
         }
+
+        if (properties.containsKey(PropertyConverter.USE_PATH_STYLE)) {
+            String value = properties.get(PropertyConverter.USE_PATH_STYLE);
+            Preconditions.checkArgument(!Strings.isNullOrEmpty(value), "use_path_style cannot be empty");
+            Preconditions.checkArgument(value.equalsIgnoreCase("true")
+                    || value.equalsIgnoreCase("false"), "Invalid use_path_style value: %s", value);
```

Comment:
"Invalid use_path_style value: %s, only 'true' or 'false' is acceptable"

Reply:
> "Invalid use_path_style value: %s, only 'true' or 'false' is acceptable"

resolved